app/views/admin/facilities/index.html.erb

```diff
@@ -2,10 +2,8 @@
   <h1 class="page-title">Facilities</h1>
 
   <div class="page-nav">
-    <% if policy([:manage, :facility, Facility]).upload? %>
+    <% if current_admin.can_manage_organizations?(Organization) %>
```

Comment:
Would this check only ever return true for super admins?

Reply:
You could have org-level permissions only on IHCI, but not on an org for a different domain (like PATH).